docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
```

Comment:
@georgkrause  Mostly this was here in case we wanted to use Swagger UI since it would allow you to easily spin up either, but if we're going down the path of Rapidoc instead it's probably not necessary.